ui/pages/index.js

```diff
@@ -4,17 +4,19 @@ import { useRouter } from 'next/router';
 
 import BuildStatus from './build-status';
 import ProjectsView from './projects';
+import Nightlies from './nightlies';
 import SecurityStatus from './security-status';
 
 export const paths = [
   { title: 'Projects', location: '/projects', component: ProjectsView },
   { title: 'Build Status', location: '/build-status', component: BuildStatus },
+  { title: 'Nigthlies', location: '/nightlies', component: Nightlies },
   { title: 'Security Status', location: '/security-status', component: SecurityStatus }
 ];
 
 const Index = () => {
   const router = useRouter();
-  const tabIndex = paths.findIndex(({ location }) => location === router.pathname);
+  const tabIndex = paths.findIndex(({ location }) => location.startsWith(router.pathname));
```

Comment:
Why?

Reply:
because it would redirect on partial matches otherwise 😕